lib/util/priv.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Util.Priv do
+  def get_priv_dir() do
+    __MODULE__ |> Application.get_application() |> Application.app_dir("priv/data/")
+  end
```

Comment:
how about making this util function do the Path.join as well. As a user, I want to call
```
Util.Priv.get_data_file_path("foo.state")
```
rather than getting the priv/data dir and joining manually

Reply:
really like that idea